src/Roots/Acorn/Bootloader.php

```diff
@@ -201,39 +197,50 @@ protected function bootHttp(ApplicationContract $app)
     {
         $kernel = $app->make(\Illuminate\Contracts\Http\Kernel::class);
         $request = \Illuminate\Http\Request::capture();
+        $time = time();
 
         $app->instance('request', $request);
         Facade::clearResolvedInstance('request');
 
         $kernel->bootstrap($request);
 
-        try {
+        add_filter('do_parse_request', function ($do_parse, \WP $wp, $extra_query_vars) use ($app, $request) {
             if (! $app->make('router')->getRoutes()->match($request)) {
-                throw new \Symfony\Component\HttpKernel\Exception\NotFoundHttpException();
+                return $do_parse;
             }
-        } catch (\Exception $e) {
-            return;
-        }
 
-        add_filter(
-            'do_parse_request',
-            fn ($do_parse, \WP $wp, $extra_query_vars) =>
-            apply_filters('acorn/router/do_parse_request', $do_parse, $wp, $extra_query_vars),
-            100,
-            3
-        );
+            return apply_filters('acorn/router/do_parse_request', $do_parse, $wp, $extra_query_vars);
+        }, 100, 3);
+
+        // Create a default route for wordpress routes to use
+        $app->make('router')
+            ->get('{any?}', fn () => response()->json(['message' => "wordpress_request_$time" ]))
+            ->where('any', '.*');
 
-        add_action('parse_request', function () use ($kernel, $request) {
-            /** @var \Illuminate\Http\Response */
+        add_action('parse_request', function () use ($time, $kernel, $request) {
+            /**
+             * @var \Symfony\Component\HttpFoundation\Response $response
+             * @see \Illuminate\Contracts\Routing\ResponseFactory
+             */
             $response = $kernel->handle($request);
 
-            if (! $response->isServerError() && $response->status() >= 400) {
+            if ($response instanceof \Symfony\Component\HttpFoundation\Response
+                && ! $response->isServerError()
+                && $response->getStatusCode() >= 400
+            ) {
                 return;
             }
 
-            $body = $response->send();
+            if (in_array(false, [
+                $response instanceof \Illuminate\Http\JsonResponse,
+                is_string($response->getContent()),
+                $data = json_decode($response->getContent()),
+                isset($data->message) && $data->message == "wordpress_request_$time",
+            ])) {
+                $body = $response->send();
 
-            $kernel->terminate($request, $body);
+                $kernel->terminate($request, $body);
+            }
```

Comment:
This `bootHttp()` method is already a bit too chonky. Can you extract this into a separate method?

You can call it like this
```php
add_action('parse_request', fn () => $this->handleLaravelRequest($time, $kernel, $request));
```
(I have no preference for what the method is called, so if you have a better name, feel free to use it.)

This arguably doesn't belong in the Bootloader at all, but that's fine for now.